/*
 * agent_checkalive.h
 *
 *  Created on: 2014Äê8ÔÂ27ÈÕ
 *      Author: Jiansong
 */

#ifndef AGENT_CHECKALIVE_H_
#define AGENT_CHECKALIVE_H_

#include<string.h>
#include<stdio.h>
#include<tinyxml.h>
#include<curl_http.h>
#include "log.h"
#include "sockaddr.h"
#include "sock_util.h"
#include <errno.h>
#include"admin.h"
#include "admin_protocol.pb.h"
#include<curl_http.h>
#include"json/json.h"
#include "config.h"

#endif /* AGENT_CHECKALIVE_H_ */
